akka-stream/src/main/scala/akka/stream/stage/Stage.scala

```diff
@@ -571,6 +573,9 @@ sealed trait Context[Out] {
    * It can be used to materialize sub-flows.
    */
   def materializer: FlowMaterializer
+
+  /** Returns operation attributes associated with the this Stage */
+  def attributes: OperationAttributes
```

Comment:
good! we’ll need to invest some effort to make things work when actually fusing stages, but this API will stay the same